server/docker/db/init/create_table.sh

```
@@ -0,0 +1,34 @@
+    name VARCHAR(50) NOT NULL,
+    email VARCHAR(100) NOT NULL UNIQUE,
+    password_hash VARCHAR(255) NOT NULL,
+    status ENUM('active', 'inactive', 'suspended') NOT NULL DEFAULT 'active',
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP NULL DEFAULT NULL,
+    INDEX idx_email (email)
+);
+
+INSERT INTO users (id, name, email, password_hash, status) VALUES
+    (1, 'John Doe', 'johndoe@example.com', '$2a$10$EXAMPLE_HASH_1', 'active'),
```

Shellcheck

[error] Braces are required for positionals over 9, e.g. ${10}.

(SC1037)